src/Components/DisplayBooks.js

```diff
@@ -1,45 +1,61 @@
-import React from 'react';
-import {ListGroupItem,ListGroup, Card, CardBody, CardHeader, CardFooter, CardTitle, CardSubtitle} from 'reactstrap';
-import StarRating from './RenderStarRating';
+import React, { lazy, Suspense } from 'react';
+import { Card, CardBody, CardFooter, CardTitle, CardSubtitle } from 'reactstrap';
+import StarRating from './StarRating';
+const DisplayBooksSecondPart = lazy(() => import('./DisplayBooksSecondPart'), 'default')
+
+function bookDisplayCards(book) {
```

Comment:
Is it possible to extract this a separate component and just use it as per requirement?

Rule of thumb: Any jsx redering logic should not be outside render method